.github/workflows/main.yml

```diff
@@ -111,7 +114,35 @@ jobs:
 
       - name: Packaging
         run: |
-          cp .buildkite_win/appxmanifest.xml kolibri-windows/
+          # When running workflows for a tag, GitHub actions/checkout@v2 doesn't
+          # just check out the tag. Instead it does the following:
+          #  - pulls the repo
+          #  - uses ref-parse to determine the commit object the tag points at
+          #  - re-fetches with 'git fetch ... origin +$COMMIT_ID:refs/tags/$TAG'
+          #  - checks out that tag
+          # As a result it loses the signed tag, which means 'git describe'
+          # reports the version as the previous tag, plus a bunch of commits.
+          # Really weird.
+          #
+          # So we can't use 'git describe' and expect to get a bare tag back
+          # when we're building that tag. Happily, we can just get the tag name
+          # from the GitHub Actions environment and use that directly.
+          if (($env:GITHUB_ACTIONS -eq "true") -and ($env:GITHUB_REF_TYPE -eq "tag")) {
+              $describe = $env:GITHUB_REF_NAME
+          } else {
+              $describe = (git describe --always)
+          }
+
+          $ver_arr = $describe -split '[-v]'
+          $ver_str = $ver_arr[1] + "." + $ver_arr[2]
+
+          echo "Set package's version as $ver_str"
+          $src = ".buildkite_win/appxmanifest.xml"
```

Comment:
I just noticed that all these files are in `.buildkite_win` but buildkite is a particular build service that we are not using. Maybe that directory should just be renamed to `kolibri-windows` and thsi file modified in place.

Reply:
We already create `kolibri-windows` folder during build.  So, it will be good to user other name.

Reply:
How about the simple `.build_win`?

Reply:
Or, we can change the folder name later.

Reply:
Later, sure.

I don't think this should be a dotfile.